kube/objects.go

```diff
@@ -78,6 +82,18 @@ func (c *Client) FetchObjects(ctx context.Context, filter ObjectFilter) (*Object
 		objects.Nodes, err = client.Nodes().List(gCtx, opts)
 		return
 	})
+	g.Go(func() (err error) {
+		objects.StorageClasses, err = storageClient.StorageClasses().List(gCtx, opts)
+		if err != nil {
+			return err
+		}
+		for _, s := range objects.StorageClasses.Items {
+			if v, ok := s.Annotations["storageclass.kubernetes.io/is-default-class"]; ok && v == "true" {
```

Comment:
nit: the check on `ok` could be dropped since a missing field would not have a value of `"true"`.